Library/Homebrew/formulary.rb

```diff
@@ -257,6 +258,7 @@ def self.load_formula_from_api(name, flags:)
       resource "ruby-source" do
         tap_git_head = json_formula.fetch("tap_git_head", "HEAD")
         ruby_source_path = json_formula.fetch("ruby_source_path", "Formula/#{name}.rb")
+        ruby_source_path = CGI.escape(ruby_source_path)
```

Comment:
```suggestion
        ruby_source_path = json_formula.fetch("ruby_source_path", "Formula/#{CGI.escape(name)}.rb")
```

Reply:
Otherwise the slash is also escaped.

Reply:
That will only escape the fallback option if `ruby_source_path` doesn't exist in the JSON formula definition. We want to escape it in both cases. Beyond that Github doesn't seem to care if you escape the slash in a url.

- https://raw.githubusercontent.com/Homebrew/homebrew-core/9fa25e536710603c675fe187c564cbaac45fb7d5/Formula%2Fbonnie%2B%2B.rb

Reply:
Ah, right.